src/open_clip/hf_configs.py

```diff
@@ -0,0 +1,15 @@
+
+
+# HF architecture dict:
+arch_dict = {
+  "roberta-base": {
```

Comment:
I think "roberta" should work as key here. All RoBERTa models should have the same config naming convention regardless the size

Reply:
Right, so how do we make it so you don't need to pass both "roberta" and "roberta-base" during initialization? You'd need the latter for AutoModel/Config and the former for our arch_dict